Init readme (#16)
thoth-station/license-solver: README.md

# License solver

Welcome to Thoth's license-solver repository

This repository serves for license detection and classifier detection from PyPI metadata.

## How it works

> license-solver detect licenses and classifiers from input files/directory. If the solver can't find a license or

Reviewer: ```suggestion
license-solver detects licenses and classifiers from input files/directory. If the solver can't find a license or
```

classifier related to the supported name, then the result is for them `UNKNOWN`, for undetected license version
is `UNDETECTED`. Some licenses do not have versions, and their naming is `LICENSE-WITHOUT-VERSION`. All dictionaries with the most used aliases are stored in thoth/license_solver/data. <br>
After successful detection, license and classifier will be compared. If they don't match, then in final JSON will be
warning: True. In case if it is missing license or classifier, comparing will be aborted. <br>
In final will be printed JSON on STDOUT.

> ## Important aliases

Reviewer: It is not directly clear to me what aliases are in this section.

Author: ```suggestion
## Important aliases
This section provides information about specific implementation detail, which can be not obviously clear.
- `BSD` license name in metadata without specific clause is `4-clause BSD` in license-solver implementation ([source](https://en.wikipedia.org/wiki/BSD_licenses#Terms))
```
it is better?

Reviewer: Sounds better 👍🏻

license name:
- `BSD` without clause = 4 clause BSD ([source](https://en.wikipedia.org/wiki/BSD_licenses#Terms))

classifier:
- None for now

## Run solver locally

> Often, it is useful to run adviser locally to experiment or verify your changes in implementation. You can do so easily

Reviewer: ```suggestion
Often, it is useful to run license-solver locally to experiment or verify your changes in implementation. You can do so easily
```
